### PR TITLE
[IMP] .gitmodules: Update account_reconcile_partial repository T#63405

### PR DESCRIPTION
**Pull request by ajqn9094**
The current repository git@git.vauxoo.com:vauxoo/account_reconcile_partial.git 
is replaced with git@git.vauxoo.com:vauxoo/account-reconcile-partial.git due
the latter complies with the naming guidelines.

### CODE REVIEW

**Comment by ajqn9094** [comment]
@luisg123v Could you review please?

Thanks in advance.

CC @desdelinux

**Comment by ajqn9094** [comment]
@luisg123v Could you review please?

CC @desdelinux

File: .gitmodules [hunk 1]
```diff
@@ -2,9 +2,9 @@
 	path = plecamer
 	url = git@git.vauxoo.com:Vauxoo/plecamer.git
 	branch = 15.0
-[submodule "account_reconcile_partial"]
+[submodule "account-reconcile-partial"]
 	path = account_reconcile_partial
```
Comment (luisg123v) [inline .gitmodules]: You didn't update it here

File: .gitmodules [hunk 2]
```diff
@@ -2,9 +2,9 @@
...
-	url = git@git.vauxoo.com:vauxoo/account_reconcile_partial.git
+	url = git@git.vauxoo.com:vauxoo/account-reconcile-partial.git
 	branch = 13.0
```
Comment (luisg123v) [inline .gitmodules]: 15

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (ajqn9094) [reply .gitmodules]: Done

File: .gitmodules (hunk 2 above)
Comment (ajqn9094) [reply .gitmodules]: Done

**Comment by ajqn9094** [comment]
@luisg123v Changes applied.

**Review by luisg123v — APPROVED** [review]

**Pull request merged**